Mantle/MTLJSONAdapter.m

```diff
@@ -356,7 +356,7 @@ - (id)modelFromJSONDictionary:(NSDictionary *)JSONDictionary error:(NSError **)e
 				NSDictionary *userInfo = @{
 					NSLocalizedDescriptionKey: [NSString stringWithFormat:@"Caught exception parsing JSON key path \"%@\" for model class: %@", JSONKeyPaths, self.modelClass],
 					NSLocalizedRecoverySuggestionErrorKey: ex.description,
-					NSLocalizedFailureReasonErrorKey: ex.reason,
+                    NSLocalizedFailureReasonErrorKey: ex.reason == nil ? @"" : ex.reason,
```

Comment:
Can I bother you with one more whitespace pass? Xcode used to pick up the tabs preference from the project file, not sure if that broke…

```suggestion
					NSLocalizedFailureReasonErrorKey: ex.reason == nil ? @"" : ex.reason,
```